Can't create container when authenticated with API token

**Describe the bug**
The provider fails on apply with errors like:
```
╷
│ Error: error creating container: received an HTTP 403 response - Reason: Permission check failed (changing feature flags for privileged container is only allowed for root@pam)
│ 
```
with no feature flags are defined in the template.

**To Reproduce**
Steps to reproduce the behavior:
1. Create an API token for root@pam user Administrator permissions, e.g. `root@pam!terraform`
2. Configure provider to authenticate with the token
3. Apply plan:
   ```hcl
   resource "proxmox_virtual_environment_container" "test_container" {
      node_name = var.virtual_environment_node_name
    
      disk {
        datastore_id = "local-lvm"
        size         = 8
      }
    
      initialization {
        hostname = "test"
    
        ip_config {
          ipv4 {
            address = "dhcp"
          }
        }
      }
    
      network_interface {
        name = "vmbr0"
      }
    
      operating_system {
        template_file_id = "local:vztmpl/ubuntu-23.04-standard_23.04-1_amd64.tar.zst"
        type             = "ubuntu"
      }
    }
   ```
4. See error
<img width="1253" alt="Screenshot 2023-10-08 at 7 49 17 PM" src="https://github.com/bpg/terraform-provider-proxmox/assets/627562/01580d3f-d297-443d-860e-65c2752d3ef8">

**Expected behavior**
A container created without errors.

**Additional context**
Add any other context about the problem here.
- Single or clustered Proxmox: both
- Provider version (ideally it should be the latest version): 0.33.0
- Terraform version: 1.5.x
- OS (where you run Terraform from): OSX
- Debug logs (`TF_LOG=DEBUG terraform apply`):
  ```
  2023-10-08T19:51:00.297-0400 [DEBUG] provider.terraform-provider-proxmox: next VM identifier: tf_req_id=dc66da0e-2468-dad7-517f-8c57230737a0 @module=proxmox tf_mux_provider=tf5to6server.v5tov6Server id=100 tf_provider_addr=registry.terraform.io/bpg/proxmox tf_resource_type=proxmox_virtual_environment_container tf_rpc=ApplyResourceChange @caller=/Users/pasha/code/terraform-provider-proxmox/proxmox/cluster/cluster.go:72 timestamp=2023-10-08T19:51:00.297-0400
  2023-10-08T19:51:00.297-0400 [DEBUG] provider.terraform-provider-proxmox: Sending HTTP Request: @module=proxmox Authorization=PVEAPIToken=root@pam!terraform=66db7035-0012-46d7-b942-xxxxxxxxxxxx Content-Type=application/x-www-form-urlencoded tf_http_trans_id=ced444a6-a8c2-f4c1-9f82-b0621047d3d2 tf_req_id=dc66da0e-2468-dad7-517f-8c57230737a0 tf_rpc=ApplyResourceChange @caller=/Users/pasha/go/pkg/mod/github.com/hashicorp/terraform-plugin-sdk/v2@v2.29.0/helper/logging/logging_http_transport.go:162 Content-Length=368 Host=pve.lab:8006 User-Agent=Go-http-client/1.1 tf_http_op_type=request tf_http_req_uri=/api2/json/nodes/pve/lxc tf_resource_type=proxmox_virtual_environment_container Accept=application/json Accept-Encoding=gzip tf_http_req_body=arch=amd64&cmode=tty&console=1&cores=1&cpuunits=1024&features=fuse%3D0%2Ckeyctl%3D0%2Cnesting%3D0&hostname=test&memory=512&net0=bridge%3Dvmbr0%2Cip%3Ddhcp%2Cname%3Dvmbr0&onboot=1&ostemplate=local%3Avztmpl%2Fubuntu-23.04-standard_23.04-1_amd64.tar.zst&ostype=ubuntu&rootfs=volume%3Dlocal-lvm%3A8&start=1&storage=local-lvm&swap=0&template=0&tty=2&unprivileged=0&vmid=100 tf_http_req_method=POST tf_http_req_version=HTTP/1.1 tf_mux_provider=tf5to6server.v5tov6Server tf_provider_addr=registry.terraform.io/bpg/proxmox timestamp=2023-10-08T19:51:00.297-0400
  2023-10-08T19:51:00.306-0400 [DEBUG] provider.terraform-provider-proxmox: Received HTTP Response: Server=pve-api-daemon/3.0 tf_http_res_status_code=403 tf_http_res_status_reason="403 Permission check failed (changing feature flags for privileged container is only allowed for root@pam)" tf_mux_provider=tf5to6server.v5tov6Server tf_req_id=dc66da0e-2468-dad7-517f-8c57230737a0 @module=proxmox Expires="Sun, 08 Oct 2023 23:51:00 GMT" Pragma=no-cache tf_rpc=ApplyResourceChange Date="Sun, 08 Oct 2023 23:51:00 GMT" tf_http_op_type=response tf_http_res_body={"data":null} tf_provider_addr=registry.terraform.io/bpg/proxmox @caller=/Users/pasha/go/pkg/mod/github.com/hashicorp/terraform-plugin-sdk/v2@v2.29.0/helper/logging/logging_http_transport.go:162 Content-Length=13 tf_http_res_version=HTTP/1.1 tf_resource_type=proxmox_virtual_environment_container tf_http_trans_id=ced444a6-a8c2-f4c1-9f82-b0621047d3d2 Cache-Control=max-age=0 Content-Type=application/json;charset=UTF-8 timestamp=2023-10-08T19:51:00.306-0400
  2023-10-08T19:51:00.306-0400 [ERROR] provider.terraform-provider-proxmox: Response contains error diagnostic: @caller=/Users/pasha/go/pkg/mod/github.com/hashicorp/terraform-plugin-go@v0.19.0/tfprotov6/internal/diag/diagnostics.go:58 diagnostic_detail= tf_req_id=dc66da0e-2468-dad7-517f-8c57230737a0 diagnostic_severity=ERROR tf_proto_version=6.4 @module=sdk.proto diagnostic_summary="error creating container: received an HTTP 403 response - Reason: Permission check failed (changing feature flags for privileged container is only allowed for root@pam)" tf_provider_addr=registry.terraform.io/bpg/proxmox tf_resource_type=proxmox_virtual_environment_container tf_rpc=ApplyResourceChange timestamp=2023-10-08T19:51:00.306-0400
  2023-10-08T19:51:00.312-0400 [DEBUG] State storage *statemgr.Filesystem declined to persist a state snapshot
  2023-10-08T19:51:00.312-0400 [ERROR] vertex "proxmox_virtual_environment_container.test_container" error: error creating container: received an HTTP 403 response - Reason: Permission check failed (changing feature flags for privileged container is only allowed for root@pam)

  ```

--- a/proxmoxtf/resource/container.go
+++ b/proxmoxtf/resource/container.go
@@ -1226,10 +1226,18 @@ func containerCreateCustom(ctx context.Context, d *schema.ResourceData, m interf
 	keyctl := types.CustomBool(featuresBlock[mkResourceVirtualEnvironmentContainerFeaturesKeyControl].(bool))
 	fuse := types.CustomBool(featuresBlock[mkResourceVirtualEnvironmentContainerFeaturesFUSE].(bool))
 
-	features := containers.CustomFeatures{
-		Nesting:    &nesting,
-		KeyControl: &keyctl,
-		FUSE:       &fuse,
+	features := containers.CustomFeatures{}
+
+	if nesting {
+		features.Nesting = &nesting
+	}
+
+	if keyctl {
+		features.KeyControl = &keyctl
+	}
+
+	if fuse {
+		features.FUSE = &fuse
 	}
 
 	initialization := d.Get(mkResourceVirtualEnvironmentContainerInitialization).([]interface{})
@@ -1567,12 +1575,7 @@ func containerCreateStart(ctx context.Context, d *schema.ResourceData, m interfa
 
 	containerAPI := api.Node(nodeName).Container(vmID)
 
-	// Start the container and wait for it to reach a running state before continuing.
-	err = containerAPI.StartContainer(ctx, 60)
-	if err != nil {
-		return diag.FromErr(err)
-	}
-
+	// A container was started by the create operation, so wait for it to be running.
 	err = containerAPI.WaitForContainerState(ctx, "running", 120, 5)
 	if err != nil {
 		return diag.FromErr(err)

